src/frontend/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import './Home.css';
+import { Navbar, HorizontalCard } from '../../components';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import { ProductionQuantityLimitsRounded } from '@mui/icons-material';
+const Home = () => {
+    const [categories, setCategories] = useState([]);
+    const [featured, setFeatured] = useState([]);
```

Comment:
You can make the context call "Data provider" which will provide the data for categories and products.

Reply:
thank you gopal for the suggestion i have made the context for data and will merge the pr